core/src/utils/test/playwright/generator.ts

```diff
@@ -27,19 +30,27 @@ interface TestConfigOption {
  * each test title is unique.
  */
 const generateTitle = (title: string, config: TestConfig): string => {
-  const { mode, direction } = config;
+  const { mode, direction, theme } = config;
 
-  return `${title} - ${mode}/${direction}`;
+  if (theme === 'light') {
+    return `${title} - ${mode}/${direction}`;
+  }
+
+  return `${title} - ${mode}/${direction}/${theme}`;
```

Comment:
Can we add a comment as to why we have this conditional? Same thing for `generateScreenshotName` below. (I know why, I'm just concerned this context will be lost over time)